core/src/main/scala/kafka/server/KafkaApis.scala

```diff
@@ -71,7 +71,7 @@ class KafkaApis(val requestChannel: RequestChannel,
   def handle(request: RequestChannel.Request) {
     try {
       trace("Handling request:%s from connection %s;securityProtocol:%s,principal:%s".
-        format(request.requestObj, request.connectionId, request.securityProtocol, request.session.principal))
+        format(request.requestDesc(false), request.connectionId, request.securityProtocol, request.session.principal))
```

Comment:
The argument of describe() is only used for old style queries and out of the last 2 still in use only Fetch makes use of it, so in the end I don't think it really matters. But for the sake of consistency, I'll change it to true